Products/Zuul/infos/metricserver.py

```diff
@@ -347,7 +350,7 @@ def _getGraphPoints(self, klass):
     def _updateRPNForMultiContext(self, infos, knownDatapointNames):
         for info in infos:
             if getattr(info._object, "rpn", False):
-                newRPN = mutateRPN(info._context.id, knownDatapointNames, info._object.rpn)
+                newRPN = mutateRPN(info._prefix, knownDatapointNames, info._object.rpn)
```

Comment:
Can you make _prefix a method call instead?  Don't like the side effect of having to have called the "id" or "name" method first in order for this to work..